README.md

```diff
@@ -60,8 +60,16 @@ $ ./sketchtool.sh "/Users/hubble/Desktop/MyDesign.sketch"
 
 # You can optionally specify an asset output dir. Otherwise current working directory will be used:
 $ ./sketchtool.sh "/Users/hubble/Desktop/MyDesign.sketch" "/var/hubble/assets/images"
+
+# You can also export to a Style Dictionary compatible format.
+$ ./sketchtool.sh "/Users/hubble/Desktop/MyDesign.sketch" --useStyleDictionaryOutput
```

Comment:
Sketchtool is not responsible for exporting tokens, this should be updated in the section above.

Reply:
Aw geesh 😞